crates/miden-lib/src/account/interface/mod.rs

```diff
@@ -186,6 +205,8 @@ impl AccountInterface {
     ///
     ///     push.{asset amount}
     ///     call.::miden::contracts::faucets::basic_fungible::distribute dropw dropw drop
+    ///
+    ///     call.::miden::contracts::auth::basic::auth__tx_rpo_falcon512
```

Comment:
We shouldn't be calling the auth procedure explicitly from anywhere (it's already always called in the epilogue)